pkg/appmanager/as3Types.go

```diff
@@ -104,6 +104,12 @@ type (
 		Class             string          `json:"class,omitempty"`
 		LoadBalancingMode string          `json:"loadBalancingMode,omitempty"`
 		Members           []as3PoolMember `json:"members,omitempty"`
+		Monitors          []as3MonitorUse `json:"monitors,omitempty"`
+	}
+
+	// as3Monitor maps to Pool in AS3 Resources
+	as3MonitorUse struct {
+		Use string `json:"use,omitempty"`
```

Comment:
No need to create a new type for this.
**as3ResourcePointer** can be used here.